src/components/molecules/Carousel/Component.tsx

```diff
@@ -34,13 +39,12 @@ const Carousel: FC<CarouselProps> = forwardRef<any, CarouselProps>(
         >
           {data.map((source, i) => (
```

Comment:
can we make it customizable too? i mean render item method ... maybe we support set molecule as a render item